src/Kanvas/Filesystem/Traits/HasFilesystemTrait.php

```diff
@@ -70,6 +71,9 @@ public function addFileFromUrl(string $url, string $fieldName): bool
         }
 
         $attachFilesystem = new AttachFilesystemAction($fileSystem, $this);
+        $app = $this->app ?? app(Apps::class);
```

Comment:
@FredPeal you dont need this, jsut need to use the method fired to execute a workflow https://github.com/bakaphp/kanvas-ecosystem-api/blob/development/src/Domains/Guild/Leads/Observers/LeadObserver.php#L70